packages/modules-sdk/src/utils/linking-error.ts

```diff
@@ -0,0 +1,24 @@
+const typeToMethod = new Map([
+  [`dismiss`, `dismiss`],
+  [`link`, `create`],
+])
+
+type LinkingErrorMessageInput = {
+  moduleA: string
+  moduleAKey: string
+  moduleB: string
+  moduleBKey: string
+  type: "dismiss" | "link"
+}
+
+/**
+ *
+ * Example: Module to dismiss salesChannel and apiKey by keys sales_channel_id and api_key_id was not found. Ensure the link exists, keys are correct, and the link is passed in the correct order to method 'remoteLink.dismiss'
+ */
+export const linkingErrorMessage = (input: LinkingErrorMessageInput) => {
```

Comment:
@carlos-r-l-rodrigues, this change OK for you?

Reply:
Just bumping this one in case you missed it 😄

Reply:
Absolutely...